frontend/src/utils/urlutils.tsx

```diff
@@ -38,6 +38,8 @@ export const MULTIPLE_MAPS_1_PARAM_KEY = 'multiple-maps'
 export const MULTIPLE_MAPS_2_PARAM_KEY = 'multiple-maps2'
 export const HIGHEST_LOWEST_GEOS_1_PARAM_KEY = 'highest-lowest-geos'
 export const HIGHEST_LOWEST_GEOS_2_PARAM_KEY = 'highest-lowest-geos2'
+export const ALT_TABLE_VIEW_1_PARAM_KEY = 'alt-table-view1'
```

Comment:
technically these shouldn't really be `_PARAM_KEY` yet because they aren't used in the url search params to record the open/closed state of the alt table... but im fine leaving them like this because we should move that state into the url bar anyways as part of #1680